keras/applications/efficientnet.py

```diff
@@ -202,6 +223,10 @@
     input_shape: Optional shape tuple, only to be specified
         if `include_top` is False.
         It should have exactly 3 inputs channels.
+    lite: Optional boolean, whether to create a 'lite' variant of the network.
+        Note, that only B0-B4 variants have lite option. For more details on
+        lite variants, you can read:
+        https://github.com/tensorflow/tpu/blob/master/models/official/efficientnet/lite/README.md
```

Comment:
They should be separate classes each. This will also make them testable.

Reply:
@fchollet Moved the models into a separate file.